packages/stylelint-config/package.json

```diff
@@ -21,16 +21,18 @@
     "stylelint-config"
   ],
   "peerDependencies": {
-    "prettier": ">= 2.0.0",
-    "stylelint": ">= 13.0.0"
+    "prettier": ">= 2.5.0",
+    "stylelint": ">= 14.3.0"
   },
   "devDependencies": {
-    "jest": "^26.6.3"
+    "jest": "^27.5.1"
   },
   "dependencies": {
-    "stylelint-config-prettier": "^8.0.1",
-    "stylelint-config-sass-guidelines": "^7.1.0",
-    "stylelint-order": "^4.1.0"
+    "postcss": "^8.4.6",
+    "stylelint": "^14.3.0",
```

Comment:
Do these need to be introduced as new dependencies? If so, why? I don't recall the reasoning, but I think we deliberately decided to only include `stylelint` as a `peerDepdencency.`

Reply:
If I run `yarn install` without those dependencies, I get unmet peer dependency warnings. But now that I think about it, they would not be unmet if they are included in the project calling this library... is that right? If that's the case, I think `"postcss": "^8.4.6"` should also be included under peerDependencies, and then these two can be removed. Am I thinking about that correctly?

![Without Dependencies](https://user-images.githubusercontent.com/31516660/153485072-f4d9fb0c-09d2-4a01-8860-1152194ff02c.png)

Reply:
Yes! That's what I'd do -- remove them from `dependencies` and keep them in `peerDependencies`. That way, the consuming application has full control over the specific version of each they'd like to use.

Reply:
Sounds good! Thank you for the input.